src/Sylius/Behat/Page/Admin/Product/UpdateSimpleProductPage.php

```diff
@@ -140,4 +177,43 @@ private function openTaxonBookmarks()
     {
         $this->getElement('taxonomy')->click();
     }
+
+    /**
+     * @param string $tabName
+     */
+    private function clickTabIfItsNotActive($tabName)
+    {
+        $attributesTab = $this->getElement('tab', ['%name%' => $tabName]);
+        if (!$attributesTab->hasClass('active')) {
+            $attributesTab->click();
+        }
+    }
+
+    /**
+     * @param string $code
+     *
+     * @return NodeElement
+     */
+    private function getImageElementByCode($code)
+    {
+        $images = $this->getElement('images');
+        $inputCode = $images->find('css', 'input[value="'.$code.'"]');
+
+        if (null === $inputCode) {
+            return null;
+        }
+
+        return $inputCode->getParent()->getParent()->getParent();
```

Comment:
Good idea, but.. I have tried without any result, could you tell mi how to do that correctly in this case?